tests/asr/test_zeroDS.py

```diff
@@ -86,30 +86,6 @@ def tearDownClass(cls) -> None:
         if os.path.exists(os.path.join(data_folder, "asr")):
             shutil.rmtree(os.path.join(data_folder, "asr"))
 
-    def test_simple_train(self):
```

Comment:
Any particular reason why we got rid of that one? 

Maybe let's remove only testing and leave the graph building as a quite good and nicely scoped "integration test"?

Reply:
wrong place (not in test_zeroDS.py). I think I moved it elsewhere